src/main/java/frc/robot/ChallengeFour.java

```diff
@@ -48,6 +51,9 @@ The values MUST be between (0.9 and -0.9) (negative is backwards).
 
     //this is the xbox controller which will be plugged into the drive laptop to control the robot
     private XboxController xController;
+    private double leftStickX = xController.getX(Hand.kLeft);
+    private double leftStickY = xController.getY(Hand.kLeft);
+    private double rightStickY = xController.getY(Hand.kRight);
```

Comment:
These values need to be constantly updated in the teleop loop. Just initialize them with 0, and then set their values each frame in the teleop loop.